tests/test_docker_splunk.py

```diff
@@ -1600,82 +1671,91 @@ def test_compose_1uf_command_start_service(self):
 
     @pytest.mark.skip(reason="Oracle is preventing automated downloads")
     def test_compose_1so_java_oracle(self):
-        # Standup deployment
-        self.compose_file_name = "1so_java_oracle.yaml"
-        self.project_name = generate_random_string()
-        container_count, rc = self.compose_up()
-        assert rc == 0
-        # Wait for containers to come up
-        assert self.wait_for_containers(container_count, label="com.docker.compose.project={}".format(self.project_name))
-        # Check ansible inventory json
-        log_json = self.extract_json("so1")
-        self.check_common_keys(log_json, "so")
-        try:
-            assert log_json["all"]["vars"]["java_version"] == "oracle:8"
-        except KeyError as e:
-            self.logger.error(e)
-            raise e
-        # Check container logs
-        output = self.get_container_logs("so1")
-        self.check_ansible(output)
-        # Check Splunkd on all the containers
-        assert self.check_splunkd("admin", self.password)
-        # Check if java is installed
-        exec_command = self.client.exec_create("so1", "java -version")
-        std_out = self.client.exec_start(exec_command)
-        assert "java version \"1.8.0" in std_out
+        if 'redhat' in platform:
+            assert 'Not supported'
```

Comment:
Are we tracking support for this?

Reply:
Just made one here: https://github.com/splunk/docker-splunk/issues/248